papas/detectors/clic/src/ClicTracker.cpp

```diff
@@ -48,30 +48,29 @@ double ClicTracker::resolution(const Track& track) const {
   double pt = track.p3().Pt();
   // matching the resmap defined above.
   double theta = fabs(track.theta()) * 180 / M_PI;
-  for (const auto& v : m_resMap) { 
+  for (const auto& v : m_resMap) {
     if (theta < v.first) return sigmaPtOverPt2(v.second.first, v.second.second, pt) * pt;
   }
   throw "tracker resolution not found";
   return 0;
 }
 
-  double ClicTracker::particleResolution(const Particle& ptc) const {
-    //not very nice due to particle being fed through into a track parameter in python
-    //will do for now
-    double pt = ptc.p3().Pt();
-    // matching the resmap defined above.
-    double theta = fabs(ptc.theta()) * 180 / M_PI;
-    for (const auto& v : m_resMap) {
-      if (theta < v.first) return sigmaPtOverPt2(v.second.first, v.second.second, pt) * pt;
-    }
-    throw "tracker particle resolution not found";
-    return 0;
-  }
-  
-  bool ClicTracker::electronAcceptance(double mag, const Track& track) const {
-    if ( track.p3().Pt() > mag && fabs(track.theta()) < m_thetaMax)
-      return rootrandom::Random::uniform(0, 1) > 0.95;
-    return false;
+double ClicTracker::particleResolution(const Particle& ptc) const {
+  // not very nice due to particle being fed through into a track parameter in python
+  // will do for now
+  double pt = ptc.p3().Pt();
+  // matching the resmap defined above.
+  double theta = fabs(ptc.theta()) * 180 / M_PI;
+  for (const auto& v : m_resMap) {
+    if (theta < v.first) return sigmaPtOverPt2(v.second.first, v.second.second, pt) * pt;
   }
-  
-}// end namespace papas
+  throw "tracker particle resolution not found";
+  return 0;
+}
+
+bool ClicTracker::electronAcceptance(double mag, const Track& track) const {
```

Comment:
Why is that part of ClicTracker? 
in python, it's part of the CLIC detector. 
The electron acceptance is not determined only by the tracker in principle

Reply:
OK, have put it back in the tracker.